apps/prairielearn/src/tests/helperDb.ts

```diff
@@ -121,6 +165,59 @@ async function setupDatabases(): Promise<void> {
   });
 }
 
+/**
+ * Runs all migrations before the given migration. Useful for testing migrations that have not yet been run in production.
+ * @param migrationName The name of the migration to run all migrations before.
+ * @param options
+ * @param options.drop Whether to drop the database before running the migrations. Default false.
+ */
+export async function runAllMigrationsBefore(
+  migrationName: string,
+  { drop = false }: { drop: boolean },
+): Promise<void> {
+  const beforeFile = migrationName.endsWith('.sql') ? migrationName : `${migrationName}.sql`;
+  await setupDatabases(
+    {
+      beforeTimestamp: extractTimestampFromFilename(beforeFile),
+      inclusiveBefore: false,
+    },
+    {
+      dropFirst: drop,
+      ignoreIfExists: true,
+    },
+  );
+}
+
+/**
+ * Runs all migrations including the given migration. Useful for testing migrations that have been run in production.
+ * @param migrationName The name of the migration to run all migrations including.
+ * @param options
+ * @param options.drop Whether to drop the database before running the migrations. Default false.
+ */
+export async function runAllMigrationsIncluding(
```

Comment:
Nit: IMO "run all including" is ambiguous (does it actually run all of them)?

I'd suggest names like `runMigrationsBefore` and `runMigrationsThrough`.

Reply:
Done.